src/scss/layout/_header.scss

```diff
@@ -1 +1,27 @@
 @use 'abstracts' as *;
+
+.site-header {
+  // header section
+  display: flex;
+  justify-content: space-between;
+  border-bottom: solid 1px #979797;
+
+  // logo link
+  &__logo {
+    display: block;
```

Comment:
This should be `display: flex;` with an `align-items: center` to line up the SVG & text. Although if you want to do that as a part of the larger header styles, I'm good with that!

Reply:
Thanks for the guidance on these!